documentation/service-rules/appropriate-body-users.md

```diff
@@ -198,7 +198,7 @@ The fail process takes place with the following steps:
 2. Fail the ECT by entering the following information:
     * The end date of the induction period
     * The number of terms carried out during the induction period
-3. When submitted the current (open) induction period is updated with the provided end date and number of terms. This closes the induction period. The 'Pass' state and induction completion date are written to the teacher's record via the TRS API.
+3. When submitted the current (open) induction period is updated with the provided end date and number of terms. This closes the induction period. The 'Failed' state and induction completion date are written to the teacher's record via the TRS API.
```

Comment:
Missed curly quotes or don't they work in MD?

Reply:
Ah so they will work once we upgrade to a version [with this commit](https://github.com/DFE-Digital/govuk-markdown/pull/41).